tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -8,8 +8,20 @@ This is a (incomplete) list of API differences between C# and GDScript.
 General differences
 -------------------
 
-As explained in the :ref:`doc_c_sharp`, C# generally uses ``PascalCase`` instead
-of the ``snake_case`` used in GDScript and C++.
```

Comment:
I can see how the old text may sound like we're telling you how to write your code, but it's not technically wrong that the [recommended style](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions) is to use PascalCase and you probably should use it in your own code too (specially when defining classes since lowercase will eventually be an error[^1]).

[^1]: https://blog.paranoidcoding.org/2022/04/11/lowercase-type-names.html

Reply:
Added a link to the style guide with a recommendation (which the page itself also includes).

> See the :ref:`doc_c_sharp_styleguide`, which we encourage you to also use for your own C# code.